book/src/kubernetes-changelog.md

```diff
@@ -3,6 +3,42 @@
 This page summarizes major CSI changes made in each Kubernetes release. For
 details on individual features, visit the [Features section](features.md).
 
+## Kubernetes 1.26
+
+### Features
+* GA
+    * Delegate fsgroup to CSI driver
+    * Azure File CSI migration
+    * vSphere CSI migration
+* Alpha
+    * Cross namespace volume provisioning
+
+## Kubernetes 1.25
+
+### Features
+* GA
+    * CSI inline volume
```

Comment:
I think we called this CSI ephemeral volumes

Reply:
Changed to "CSI ephemeral inline volumes":
https://kubernetes-csi.github.io/docs/ephemeral-local-volumes.html#csi-ephemeral-inline-volumes